scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
@jdsn Why would it make sense to use `crowbar_api_request` when we can simply use `--timeout` and avoid having to construct low-level API calls??

@jsuchome Sorry but I agree with @MaximilianMeister.  It takes seconds to add `--timeout 300` to an existing `crowbarctl` - is there really any good reason not to do this???  The underlying bug is non-trivial and could take a long time to fix - in the mean time I think we should aim for workarounds which are clean (not bypassing the standard `crowbarctl` code path) and easy to understand (not using low-level APIs).  If I'm missing something here then please let me know, otherwise let's go for the most obvious solution!